### PR TITLE
Updated handleError arguments to be pointers for video endpoint

### PR DESCRIPTION
**Pull request by camrice**
Starting PR here for code review. Will set to upstream after review.

### CODE REVIEW

**Comment by camrice** [comment]
For context, the labels for handling an error case were not being updated due to the `handleError` function not using a pointer. Every request was then being recorded as OK. The http writer and analytics objects were also changed to pointers. The `ao` argument is recorded the same way as the labels so it was also missing updates. I am assuming the http writer should also be a pointer, but let me know if this should change.

File: endpoints/openrtb2/video_auction.go [hunk 1]
```diff
@@ -238,7 +238,7 @@ func cleanupVideoBidRequest(videoReq *openrtb_ext.BidRequestVideo, podErrors []P
 	return videoReq
 }
 
-func handleError(labels pbsmetrics.Labels, w http.ResponseWriter, errL []error, ao analytics.AuctionObject) {
+func handleError(labels *pbsmetrics.Labels, w *http.ResponseWriter, errL []error, ao *analytics.AuctionObject) {
```
Comment (camrice) [inline endpoints/openrtb2/video_auction.go]: For context, the labels for handling an error case were not being updated due to the `handleError` function not using a pointer. Every request was then being recorded as OK. The http writer and analytics objects were also changed to pointers. The `ao` argument is recorded the same way as the labels so it was also missing updates. I am assuming the http writer should also be a pointer, but let me know if this should change.

Comment (camrice) [inline endpoints/openrtb2/video_auction.go]: For context, the labels for handling an error case were not being updated due to the `handleError` function not using a pointer. Every request was then being recorded as OK. The http writer and analytics objects were also changed to pointers. The `ao` argument is recorded the same way as the labels so it was also missing updates. I am assuming the http writer should also be a pointer, but let me know if this should change.

**Review by VeronikaSolovei9 — APPROVED** [review]

**Comment by camrice** [comment]
PR was merged to upstream

**Pull request closed without merging**